### PR TITLE
Center DialPad backspace icon

### PR DESCRIPTION
**Pull request by anjulgarg**
# What
Center DialPad backspace icon

# Why

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

**Review by dmceachernmsft — APPROVED** [review]

**Review by mgamis-msft — APPROVED** [review]

File: packages/react-components/src/components/styles/Dialpad.styles.ts [hunk 1]
```diff
@@ -95,7 +95,11 @@ export const iconButtonStyles = (theme: Theme): IButtonStyles => {
       color: `${theme.palette.black}`
     },
     icon: {
-      height: 'auto'
+      height: 'auto',
+      // Needed to keep the icon vertically centered.
+      '> span': {
+        display: 'flex'
```
Comment (edwardlee-msft) [inline packages/react-components/src/components/styles/Dialpad.styles.ts:101]: I'm surprised this is working as the span inside the icon button already has a display flex type
image

Comment (edwardlee-msft) [reply packages/react-components/src/components/styles/Dialpad.styles.ts:101]: I did find that that changing the '> i': { fontSize: '20px' } resolves this issue.

Comment (edwardlee-msft) [reply packages/react-components/src/components/styles/Dialpad.styles.ts:101]: I did find doing the following fixed the issue:
image

**Review by carocao-msft — APPROVED** [review]

File: packages/react-components/src/components/styles/Dialpad.styles.ts (hunk 1 above)
Comment (anjulgarg) [reply packages/react-components/src/components/styles/Dialpad.styles.ts:101]: :o
Strange. That explains why the snapshots were not generated

Comment (edwardlee-msft) [reply packages/react-components/src/components/styles/Dialpad.styles.ts:101]: nit: instead of using px, use 1.25rem.

**Pull request merged**